d4s2_api/utils.py

```diff
@@ -48,9 +48,12 @@ def make_delivery_message(self, accept_url):
         templates = self.delivery_details.get_action_template_text('delivery')
         return self._make_message(templates, accept_url=accept_url)
 
-    def make_processed_message(self, process_type, warning_message=''):
+    def make_processed_message(self, process_type, warning_message='', direction=MessageDirection.ToRecipient):
```

Comment:
A processed message is, by definition,ToSender`. This function should not take direction as an argument, and it should always call `_make_message` with `ToSender`.

The original (removed in fd9bb8f203e509e64ae7fdbc013317e061e6401e) class for ProcessedMessage documents this:

https://github.com/Duke-GCB/D4S2/blob/c70aa94aaccc11da3c3027aa6738a651b2509ffe/d4s2_api/utils.py#L108-L115

It looks like this distinction was lost in the refactor, the tests for that class checked the directions too:

https://github.com/Duke-GCB/D4S2/blob/c70aa94aaccc11da3c3027aa6738a651b2509ffe/d4s2_api/tests_utils.py#L76-L83

Reply:
Currently there are two messages sent when a S3 delivery is processed.
`accepted` sent to the sender and `accepted_recipient` sent to the recipient.
https://github.com/Duke-GCB/D4S2/blob/0d3c200eff0f0291da21d4155ee4f657a23c233d/switchboard/s3_util.py#L403-L426

Reply:
@dleehr Any more changes requested for this?

Reply:
Let's remove the default value of `direction` from `make_processed_message()`. I think it's clear there's no obvious default direction anymore. That default seems to have caused this confusing behavior, and I think it's misleading.

The bug was that processed messages were going ToRecipient instead of ToSender. This PR fixes that bug and writes tests to confirm that fix, so 👍.